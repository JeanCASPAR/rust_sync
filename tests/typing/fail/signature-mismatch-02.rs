use rust_sync::sync;

sync! {
    #![pass(1)]
    
    node oui(a : int) = (a);

    node non() = ()
    where
        o : int = oui();
}

fn main() {}
