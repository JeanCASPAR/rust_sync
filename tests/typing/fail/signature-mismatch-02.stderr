error: node `oui` takes 1 argument, but 0 were provided
  --> tests/typing/fail/signature-mismatch-02.rs:10:19
   |
10 |         o : int = oui();
   |                   ^^^^^
